pages/index.js

```diff
@@ -3,10 +3,16 @@ import { Inter } from "next/font/google";
 import styles from "@/styles/Home.module.css";
 import UserForm from "@/components/User/IssueForm/UserForm";
 import IssueContextProvider from "@/utils/IssueContext";
+import { signIn, signOut, useSession } from "next-auth/react";
+import { getServerSession } from "next-auth";
+import { AuthOptions } from "next-auth";
 
 const inter = Inter({ subsets: ["latin"] });
 
 export default function Home() {
+  const { data: session, status } = useSession();
```

Comment:
This is form right? No user session is required on this page? 

Maybe remove this? and auth related things?

Reply:
ok, cleaner code now :)